src/tests/components/App.test.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import axios from "axios";
+import { render, screen, waitFor } from "@testing-library/react";
+import App from "../../components/App";
+import forecast from "../../data/forecast.json";
+
+// describe("App", () => {
+//   test("renders the App component correctly", () => {
+//     render(<App location={forecast.location} forecasts={forecast.forecasts} />);
+//     const h1Element = screen.getByText(/Manchester, UK/i);
+//     expect(h1Element).toBeInTheDocument();
+//   });
+// });
+
+describe("App", () => {
+  test("renders the App component correctly", async () => {
```

Comment:
In this file you've called you've used `test`, but in your other files you've used `it`. It doesn't matter which you use, but you should be consistent with your use throughout